app/assets/src/components/BulkUploadImport.jsx

```diff
@@ -32,6 +39,8 @@ class BulkUploadImport extends React.Component {
     this.userDetails = props.loggedin_user;
     this.toggleCheckBox = this.toggleCheckBox.bind(this);
     this.openCliModal = this.openCliModal.bind(this);
+    this.bulkPathRef = null;
```

Comment:
Using string refs is no longer recommended :D So we need to use a callback function to set ref on the element we want a ref for.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-string-refs.md